### ISSUE TITLE
salmon: error while loading shared libraries: libtbb.so.2: cannot open shared object file: No such file or directory

### ISSUE THREAD

**Issue by ojziff** [TaliaferroLab/LABRAT#3]
Hi @taliaferrojm, thanks for this terrific tool!

Whilst running labrat in runSalmon mode I am getting error:

`salmon: error while loading shared libraries: libtbb.so.2: cannot open shared object file: No such file or directory
`

The same error has been reported with bowtie2: https://www.metagenomics.wiki/tools/bowtie2/install/libtbb-so-2

I have tried to address this by installing tbb with `conda install -c bioconda tbb` and by creating a dedicated labrat environment with the labratenv.yml but neither of these are working.

Please could you advise how i can overcome this error.

These are the packages in my labrat conda env:

```
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                        main
_openmp_mutex             4.5                       1_gnu
argcomplete               1.12.3             pyhd3eb1b0_0
argh                      0.26.2                   py36_0
biopython                 1.78             py36h7b6447c_0
blas                      1.0                         mkl
bzip2                     1.0.8                h7b6447c_0
ca-certificates           2021.5.25            h06a4308_1
certifi                   2021.5.30        py36h06a4308_0
gffutils                  0.9                        py_1    bioconda
importlib-metadata        3.10.0           py36h06a4308_0
importlib_metadata        3.10.0               hd3eb1b0_0
intel-openmp              2021.2.0           h06a4308_610
labrat                    0.2.2                    pypi_0    pypi
ld_impl_linux-64          2.35.1               h7274673_9
libffi                    3.3                  he6710b0_2
libgcc-ng                 9.3.0               h5101ec6_17
libgfortran-ng            7.5.0               ha8ba4b0_17
libgfortran4              7.5.0               ha8ba4b0_17
libgomp                   9.3.0               h5101ec6_17
libstdcxx-ng              9.3.0               hd4cf53a_17
mkl                       2020.2                      256
mkl-service               2.3.0            py36he8ac12f_0
mkl_fft                   1.3.0            py36h54f3939_0
mkl_random                1.1.1            py36h0573a6f_0
ncurses                   6.2                  he6710b0_1
numpy                     1.19.2           py36h54aff64_0
numpy-base                1.19.2           py36hfa32c7d_0
openssl                   1.1.1k               h27cfd23_0
pandas                    1.0.5            py36h0573a6f_0
patsy                     0.5.1                    py36_0
pip                       21.1.3           py36h06a4308_0
pyfaidx                   0.6.0              pyh5e36f6f_0    bioconda
python                    3.6.13               h12debd9_1
python-dateutil           2.8.1              pyhd3eb1b0_0
pytz                      2021.1             pyhd3eb1b0_0
readline                  8.1                  h27cfd23_0
salmon                    0.14.2               ha0cc327_0    bioconda
scipy                     1.3.2            py36h7c811a0_0
setuptools                52.0.0           py36h06a4308_0
simplejson                3.8.1                    py36_0    bioconda
six                       1.16.0             pyhd3eb1b0_0
sqlite                    3.36.0               hc218d9a_0
statsmodels               0.10.1           py36hdd07704_0
tbb                       2021.2.0             hff7bd54_0
tk                        8.6.10               hbc83047_0
typing_extensions         3.10.0.0           pyh06a4308_0
wheel                     0.36.2             pyhd3eb1b0_0
xz                        5.2.5                h7b6447c_0
zipp                      3.4.1              pyhd3eb1b0_0
zlib                      1.2.11               h7b6447c_3
```

Many thanks!
Oliver

**Comment by ojziff** [comment]
Fixed by downgrading TBB to version 2020.2 with `conda install -c bioconda tbb=2020.2`
#https://github.com/COMBINE-lab/salmon/issues/565
#https://github.com/ngs-docs/2020-ggg-201b-rnaseq/commit/b61d5176118c08db2924a5622222be53edebc360

**Comment by taliaferrojm** [comment]
Thanks for bringing this up and then finding a solution. Please do not hesitate to let me know if you run into any more problems.
